src/main/java/com/example/Library/Management/System/Mapper/LibraryMapper.java

```diff
@@ -0,0 +1,43 @@
+package com.example.Library.Management.System.Mapper;
```

Comment:
follow naming conventions
```suggestion
package com.example.Library.Management.System.mapper;
```